packages/flutter_tools/test/integration.shard/gen_l10n_test.dart

```diff
@@ -180,4 +182,52 @@ void main() {
     final StringBuffer stdout = await runApp();
     expectOutput(stdout);
   });
+
+  // Analyze generated l10n file
+  // (see https://github.com/flutter/flutter/issues/137313)
+  testUsingContext('generated l10n file without problems', () async {
+    if (!fileSystem.file('${tempDir.path}/pubspec.yaml').existsSync()) {
+      await project.setUpIn(tempDir);
+    }
+    await globals.processManager.run(
+      <String>['flutter', 'gen-l10n'],
```

Comment:
i think expecting `flutter` to be on the path is the wrong thing to do here. Please use the FlutterTestDriver like the other tests in this file.